backend/db/schema.sql

```diff
@@ -0,0 +1,85 @@
+CREATE TABLE "Users" (
+	"userId" serial PRIMARY KEY,
+	"username" varchar(120) NOT NULL UNIQUE,
+	"email" varchar(255) NOT NULL UNIQUE,
+	"fullName" varchar(255),
+	"avatarPath" varchar(255),
+	"location" varchar(80),
+	"company" varchar(120),
+	"website" varchar(255),
+	"bio" text,
+	"isAdmin" boolean DEFAULT FALSE,
```

Comment:
better use role based permission system, far more flexible then isAdmin

Reply:
we'll add it later